syft/generic/pointers/pointer_plan.py

```diff
@@ -83,6 +83,21 @@ def __call__(self, *args, **kwargs):
         Transform the call on the pointer in a request to evaluate the
         remote plan
         """
+        if len(self._locations) > 1 and isinstance(args[0], sy.MultiPointerTensor):
+            responses = {}
+            for location in self._locations:
+                child_args = list(
```

Comment:
I've always found list comprehensions clearer and more pythonic

Reply:
This makes sense!